server/views/pages/vary/caseload.njk

```diff
@@ -21,6 +21,11 @@
 {% set licences = [] %}
 
 {% for offender in caseload %}
+    {% if offender.licenceStatus === "REVIEW_NEEDED" %}
+        {% set releaseDate = '<span class="approval-overdue">' + offender.releaseDate + '</span><span class="approval-overdue approval-overdue-message"><br>Timed out</span>' %}
```

Comment:
Are we always applying the `approval-overdue` class when it's timed out? Or only after a certain period?

Reply:
For now we are applying for all licences with `REVIEW_NEEDED` status.
May be if needed we can add conditions while implementing `immediate release`